src/nextjournal/clerk.clj

```diff
@@ -339,7 +339,7 @@
   (when (seq watch-paths)
     (println "Starting new watcher for paths" (pr-str watch-paths))
     (reset! !watcher {:paths watch-paths
-                      :watcher (apply beholder/watch #(file-event %) watch-paths)}))
+                      :watcher (apply beholder/watch file-event watch-paths)}))
```

Comment:
`file-event` was wrapped so it would pick up changes at the REPL without requiring a restart.

Reply:
There is no need for that.

Reply:
```
user=> (defn foo [] :foo)
#'user/foo
user=> (defn bar [] (apply foo nil))
#'user/bar
user=> (bar)
:foo
user=> (defn foo [] :bar)
#'user/foo
user=> (bar)
:bar
```